pac4j-cas/src/test/java/org/pac4j/cas/client/CasClientTests.java

```diff
@@ -127,7 +127,7 @@ public void testBackLogout() {
         final MockWebContext context = MockWebContext.create().addRequestParameter(CasConfiguration.LOGOUT_REQUEST_PARAMETER, LOGOUT_MESSAGE)
             .setRequestMethod("POST");
         TestsHelper.expectException(() -> casClient.getCredentials(context), HttpAction.class, "back logout request: no credential returned");
-        assertEquals(200, context.getResponseStatus());
+        assertEquals(204, context.getResponseStatus());
```

Comment:
HTTP_NO_CONTENT is probably a safe one to add to CAS but let me ask: why 204? The change makes sure the logout endpoint produces a 204? for CAS?